### PR TITLE
content: clarify review process

### PR DESCRIPTION
**Pull request by shenriotpro**

### CODE REVIEW

**Review by Khoyo — APPROVED** [review]

File: content/docs/guides/contribute/review-process.fr.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ sequenceDiagram
   
   loop Boucle entre auteur et reviewer
     R-->>A: Commente, demande des changements
-    A-->>R: Répond à chaque commentaire/requested changes
+    A-->>R: Répond à chaque commentaire/demande de changement
     A-->>R: Corrige le code si nécessaire dans des « fixups » dédiés
-    R-->>A: Vérifie et commente à nouveau le code
-    R-->>A: Résoud les conversations/requested changes le cas échéant
+    R-->>A: Vérifie, teste, et commente à nouveau le code
+    R-->>A: Résout les conversations/demandes de changement le cas échéant
```
Comment (axrolld) [inline content/docs/guides/contribute/review-process.fr.md:18]: 👀

**Review by axrolld — APPROVED** [review]
Thanks for the quick fix !

File: content/docs/guides/contribute/review-process.en.md [hunk 2]
```diff
@@ -11,10 +11,10 @@ sequenceDiagram
   R->>A: Answers yes or no
   
   loop Loop between author and reviewer
-    R-->>+A: Comments, asks for changes
+    R-->>A: Comments, asks for changes
```
Comment (axrolld) [inline content/docs/guides/contribute/review-process.en.md:14]: I just don't really know what this fix is for

Comment (shenriotpro) [reply content/docs/guides/contribute/review-process.en.md:14]: I believe it's responsible for this box 
image

I removed the box in order to get consistency across languages (and I don't think the box is necessary), but I could change the other way around if needed.

**Review by clarani — APPROVED** [review]
LGTM :white_check_mark:

**Pull request merged**